repos/fdbt-site/src/pages/home.tsx

```diff
@@ -82,6 +83,23 @@ export const getServerSideProps = (ctx: NextPageContextWithSession): {} => {
         redirectTo(ctx.res, '/multipleOperators');
     }
 
+    if (sessionNoc) {
+        const services = await getAllServicesByNocCode(sessionNoc);
+        const hasBodsServices = services.some((service) => service.dataSource && service.dataSource === 'bods');
+        const tndsServices = services.filter((service) => service.dataSource && service.dataSource === 'tnds');
+
+        if (!hasBodsServices && tndsServices.length > 0) {
+            const modes = tndsServices
+                .map((service) => {
+                    return service.mode ? service.mode : '';
+                })
+                .filter((mode) => mode !== '');
```

Comment:
We need to not set the attribute if there are no modes, because otherwise we will be setting an empty array in the session.

Reply:
done